app/assets/stylesheets/variables/_bootstrap.scss

```diff
@@ -152,9 +153,9 @@ $line-height-base:           1.3 !default;
 
 // $btn-font-weight:                normal
 
-$btn-default-color:                 #666 !default;
-$btn-default-bg:                    $faint-gray !default;
-$btn-default-border:                $faint-gray !default;
+$theme-colors: (
+  "secondary": $faint-gray,
+);
```

Comment:
Hmm what is this about... need to discuss this...

Reply:
Updated